frontend/src/app/components/LocalizedHtml.js

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
Yeah, it took me weeks to figure out how to do this properly. I should write some docs. Hopefully this component can get merged back into fluent-react.